src/index.ts

```diff
@@ -225,21 +226,6 @@ export class AcrolinxEndpoint {
     };
   }
 
-  public async getTelemetryInstruments(accessToken: AccessToken): Promise<Instruments | undefined> {
```

Comment:
This was introduced in 2.x beta. Its not a breaking change. We are yet to finalize v2